### ISSUE TITLE
 Chronograf (1.7.17) Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by Clausewitz45** [influxdata/chronograf#5411]
What browser are you using (name and version)?
Google Chrome - Version 80.0.3987.132 (Official Build) (64-bit)

What operating system are you using?
Windows 10 Enterprise (sorry, I'm enforced to use this)

Please describe what you were trying to do when you encountered this error:
I was just clicking throug Chronograf, and on the main page, for News Feed I got this. 
```
TypeError: Cannot read property 'name' of undefined
    at https://monitoring-int.emnos.intra/src.f9bc0d55.js:2363:385
    at Array.map (<anonymous>)
    at l (https://monitoring-int.emnos.intra/src.f9bc0d55.js:2363:310)
    at sa (https://monitoring-int.emnos.intra/src.f9bc0d55.js:717:55869)
    at $a (https://monitoring-int.emnos.intra/src.f9bc0d55.js:717:68127)
    at oo (https://monitoring-int.emnos.intra/src.f9bc0d55.js:717:91055)
    at uo (https://monitoring-int.emnos.intra/src.f9bc0d55.js:717:91442)
    at Yo (https://monitoring-int.emnos.intra/src.f9bc0d55.js:717:98524)
    at $o (https://monitoring-int.emnos.intra/src.f9bc0d55.js:717:97904)
    at jo (https://monitoring-int.emnos.intra/src.f9bc0d55.js:717:96925)
```

**Comment by russorat** [comment]
@Clausewitz45 thanks! I think we have had similar reports about the newsfeed, so we will take a look.

**Comment by outcast** [comment]
I am having the same issue. 

What browser are you using (name and version)?
Chrome - Version 80.0.3987.122 (Official Build) (64-bit)

What operating system are you using?
Browser - OS X 10.15.3
Chronograf Server - Ubuntu 18.04


Please describe what you were trying to do when you encountered this error:

```
TypeError: Cannot read property 'name' of undefined
    at http://192.168.64.3:8888/src.f9bc0d55.js:2363:385
    at Array.map (<anonymous>)
    at l (http://192.168.64.3:8888/src.f9bc0d55.js:2363:310)
    at sa (http://192.168.64.3:8888/src.f9bc0d55.js:717:55869)
    at $a (http://192.168.64.3:8888/src.f9bc0d55.js:717:68127)
    at oo (http://192.168.64.3:8888/src.f9bc0d55.js:717:91055)
    at uo (http://192.168.64.3:8888/src.f9bc0d55.js:717:91442)
    at Yo (http://192.168.64.3:8888/src.f9bc0d55.js:717:98524)
    at $o (http://192.168.64.3:8888/src.f9bc0d55.js:717:97904)
    at eu (http://192.168.64.3:8888/src.f9bc0d55.js:717:99279)
```

**Comment by glinton** [comment]
This should be fixed by #5420 and will be available in 1.8.1 (or is available immediately with a [nightly build](https://portal.influxdata.com/downloads/)). Comment if that is not the case and we'll re-open this)
